wpilib/wpilib/analogtrigger.py

```diff
@@ -38,7 +40,7 @@ class AnalogTrigger(SendableBase):
 
     AnalogTriggerType = AnalogTriggerOutput.AnalogTriggerType
```

Comment:
You should be able to fix the cyclic import by moving this here.

Reply:
Moving what?